CODING_STANDARD.md

```diff
@@ -81,10 +81,15 @@ Formatting is enforced using clang-format. For more information about this, see
 - The priority of documentation is readability. Therefore, feel free to use
   Doxygen features, or to add whitespace for multi-paragraph comment blocks if
   necessary.
-- A comment block should immediately precede the definition of the entity it
-  documents, which will generally mean that it will live in the source file.
-  This allows us to take advantage of the one definition rule. If each entity
-  is defined only once, then it is also documented only once.
+- For functions or methods that provide an interface to a module, a comment
+  block should immediately precede the declaration of the entity
+  it documents. This implies that it will live in the header file.
+  The documentions should focus on the externally visible behavior, and
```

Comment:
Duplicating/splitting documentation doubles the effort of maintaining it and increases the likelihood of outdated documentation.
Does doxygen actually merge these documentations properly?

Reply:
Yes, doxygen does merge these; thus, the documentation in the header should not be repeated.

Reply:
What are `documentions`?